tests/test_project_time.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import shutil
+from pathlib import Path
+
+import pyspark.sql.functions as F
+import pytest
+
+import pandas as pd
+import numpy as np
+
+from dsgrid.dimension.base_models import DimensionType
+from dsgrid.registry.registry_manager import RegistryManager
+
+REGISTRY_PATH = (
+    Path(__file__).absolute().parents[1]
+    / "dsgrid-test-data"
+    / "filtered_registries"
+    / "simple_standard_scenarios"
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def registry_mgr():
+    return RegistryManager.load(REGISTRY_PATH, offline_mode=True)
+
```

Comment:
I think we need at least two more tests:
1. Verification that the TEMPO annual load data values are correct (not just time).
2. Verification that a query asking for comstock + resstock + tempo works. This can be a simple adjustment to one query in tests/test_query.py.

Reply:
Need help with 2

Reply:
Link to test query: https://github.com/dsgrid/dsgrid/blob/main/tests/test_queries.py#L986

For #2, import code from test_project_time to get tempo data with fully exploded time. in line 1065